ubuntu_desktop_installer/lib/main.dart

```diff
@@ -42,6 +42,8 @@ class UbuntuDesktopInstallerApp extends StatelessWidget {
         locale: value,
         onGenerateTitle: (context) => AppLocalizations.of(context).appTitle,
         theme: yaruLightTheme,
+        darkTheme: yaruDarkTheme,
+        themeMode: ThemeMode.system,
```

Comment:
According to [the documentation](https://api.flutter.dev/flutter/material/MaterialApp/themeMode.html), `ThemeMode.system` is the default value, so it's probably not necessary to set it here?

Reply:
Thanks, good catch :)